pkg/marker/marker.go

```diff
@@ -0,0 +1,142 @@
+// Copyright 2019 Red Hat, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package marker
+
+import (
+	"encoding/json"
+	"fmt"
+	"strings"
+
+	"github.com/golang/glog"
+	"github.com/vishvananda/netlink"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	types "k8s.io/apimachinery/pkg/types"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/rest"
+)
+
+var (
+	clientset kubernetes.Interface
+)
+
+const (
+	// Expose available bridges as resources in format bridge.network.kubevirt.io/[bridge name]
+	resourceNamespace = "bridge.network.kubevirt.io"
+	// Kubernetes API does not support infinite resources, assume that 1000 connections is enough
+	resourceDefaultValue = "1000"
+	ifcDir               = "/sys/class/net/"
```

Comment:
these two consts are not used anymore

Reply:
I prefer to do another PR for this (this or next sprint)